src/content/tools/pub/cmd/pub-token.md

```diff
@@ -3,19 +3,18 @@ title: dart pub token
 description: Manage authentication tokens for package repositories.
 ---
 
-_pub_ _token_ is one of the subcommands of the [pub command](/tools/pub/cmd).
+The `dart pub token` subcommand manages a store of secret tokens to
+authenticate against third-party servers when [publishing](pub-lish) packages
+and [retrieving](pub-get) dependencies.
 
-It is used to manage a store of secret tokens for authenticating
-against third-party servers when [publishing](pub-lish) packages and
-[retrieving](pub-get) dependencies.
-
-The tokens are stored in a
-[user-wide config dir]({{site.repo.dart.org}}/cli_util/blob/71ba36e2554f7b7717f3f12b5ddd33751a4e3ddd/lib/cli_util.dart#L88-L118). 
+It stores these tokens in a [user-wide config directory][config-dir].
 
 It has three subcommands: `add`, `list` and `remove`.
```

Comment:
suggestion (lite): This gets tricky with avoiding passive voice, but could we say something besides "It" again here (the previous sentence also starts with "It")

- `dart pub token` has three subcommands....
- The subcommand has three subcommands of its own...

I don't have any good ideas, just a thought.